.github/workflows/build.yml

```diff
@@ -20,10 +21,31 @@ jobs:
           if [[ ${#err} != 0 ]]; then
             echo "$err" | while IFS= read -r file; do
               echo "::error file={"$file"},title={checking}::Formatted $file"
-            done
-            exit 1
+              done
+              exit 1
```

Comment:
what happened here?

Reply:
eh, it's my linter reformat the file.

Reply:
fixed in https://github.com/pq-code-package/mlkem-c-aarch64/pull/29/commits/8e876910eb78296ab887f26ae78af8b6e9cc26e6